.vscode/launch.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Is this file meant to be committed?

Reply:
It is not… 🙃  https://github.com/Financial-Times/n-teaser/pull/146/commits/2197f2a124e3a760fa59ab10220b1c75fb3beda4